models/src_shopify.yml

```diff
@@ -407,9 +410,22 @@ sources:
           - name: updated_at
             description: The date and time when the product variant was last modified. Gets returned in ISO 8601 format.
           - name: weight
-            description: The weight of the product variant in the unit system specified with weight_unit.
-          - name: weight_unit
-            description: "The unit of measurement that applies to the product variant's weight. If you don't specify a value for weight_unit, then the shop's default unit of measurement is applied. Valid values: g, kg, oz, and lb."
+            description: (DEPRECATED 2025-01-06) The weight of the product variant in the unit system specified with weight_unit.
+          - name: (DEPRECATED 2025-01-06) weight_unit
+            description: >
+              The unit of measurement that applies to the product variant's weight. If you don't specify a value for weight_unit, then the shop's default unit of measurement is applied. Valid values: g, kg, oz, and lb.
```

Comment:
```suggestion
          - name: weight_unit
            description: >
              (DEPRECATED 2025-01-06)  The unit of measurement that applies to the product variant's weight. If you don't specify a value for weight_unit, then the shop's default unit of measurement is applied. Valid values: g, kg, oz, and lb.
```

Reply:
Oh boy thank you. I updated and also regenned the docs.